src/App.js

```diff
@@ -1,21 +1,47 @@
-import React from "react";
+// {useState} below allows to avoid typing "React.useState" when creating a variable
+import React, { useState } from "react";
 import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 
+// array of to do list items (used to be in TodoList.js)
+const todos = [
+  // {
+  //   id: 1,
+  //   title: "Complete assignment",
+  // },
+  // {
+  //   id: 2,
+  //   title: "Learn some React baby",
+  // },
+  // {
+  //   id: 3,
+  //   title: "Take a job-well-done nap?",
+  // },
+];
+
 // APP COMPONENT / FUNCTION//////////////////////////////////////////////////////
 function App() {
-  // create a new state variable named `newTodo` with update function named `setNewTodo` (with useState hook)
-  const [newTodo, setNewTodo] = React.useState("");
+  // REMOVED BELOW newTodo STATE VARIABLE FOR LESSON 1-4:
+  // Below sets new to do's. Creates a new state variable named `newTodo` with update function named `setNewTodo` (with useState hook)
+  // const [newTodo, setNewTodo] = useState("");
+
+  // Below stores ALL to do's. Creates new state variable called "todoList" w/"setter" called "setTodoList" w/"todos" array as default value. This was "lifting state" step.
+  const [todoList, setTodoList] = useState(todos);
```

Comment:
Thanks, i tried that and it worked, but i'm confused how...

In App.js, line 44, what are each of these parts in this code? `<TodoList todos={todoList} />`

I.e., is `todos` the term to be used in TodoList.js to refer to this prop, & `todoList` is the thing in App.js that is called via props in TodoList.js?

Reply:
I'm asking because i'm wondering how i can comment out all of `todos` in App.js (the suggestion below in your below comment), and why that didn't that break the code where i'm calling `todos` as a destructured prop in TodoList.js? Does that question make sense?